Makefile

```diff
@@ -63,6 +63,10 @@ mock:
 		-destination ./protocol/transport/mock/whisper_mock.go \
 		-package transport_mock \
 		github.com/status-im/status-console-client/protocol/transport WhisperTransport
+	mockgen \
+		-destination ./protocol/transport/srvmock/server_mock.go \
```

Comment:
```suggestion
		-destination ./protocol/transport/mock/server_mock.go \
```

Reply:
Maybe let's keep one `mock/` directory for now?

Reply:
there will be circular import this way. i am using server mock in unit tests for transport and transport_mock.go imports transport.WhisperServiceKeysManager

Reply:
So maybe just put it directly into `whisper_service_mock.go` on the same level? `svrmock` looks odd because I can't easily find where `Server` is declared. 

Another alternative would be to put it into `protocol/transport/internal`.

Reply:
it makes sense to add it to the same module as a unit test :+1: